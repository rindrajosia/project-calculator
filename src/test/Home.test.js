import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('Check if id: home-page exist', () => {
  render(<Home />);
  const home = document.querySelector('#home-page');
  expect(home).not.toBeNull();
});

it('renders correctly', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
