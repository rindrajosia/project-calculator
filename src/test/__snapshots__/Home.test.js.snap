// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="home-page"
>
  <h1>
    Welcome to our page
  </h1>
  <p>
    Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in laying out print, graphic or web designs. The passage is attributed to an unknown typesetter in the 15th century who is thought to have scrambled parts of Cicero's De Finibus Bonorum et Malorum for use in a type specimen book.
  </p>
  <p>
    Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in laying out print, graphic or web designs. The passage is attributed to an unknown typesetter in the 15th century who is thought to have scrambled parts of Cicero's De Finibus Bonorum et Malorum for use in a type specimen book.
  </p>
  <p>
    Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in laying out print, graphic or web designs. The passage is attributed to an unknown typesetter in the 15th century who is thought to have scrambled parts of Cicero's De Finibus Bonorum et Malorum for use in a type specimen book.
  </p>
</div>
`;
